testbot.py

```diff
@@ -160,4 +165,46 @@ async def join_game(arguments: list, message: discord.message):
     logging.info(f'Assigned {message.author} to {game_name}')
     await message.channel.send(f'Have fun playing {game_name}, {message.author}!')
 
+
+async def make_poll(message: discord.message):
+    command = message.content.replace("pb poll", '').strip()
+
+    if not command or "\"" not in command or ',' not in command:
+        logging.debug('No poll title or arguments were passed')
+        await message.channel.send('You can\'t just make a poll without a title or arguments! Please insert a poll '
```

Comment:
If you're showing them how to format, how about use an embed message? You can use a code field to give an example:

> Example usage:
> `pb poll "title" option1, option2, option3`